testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/DockerContainer.java

```diff
@@ -160,6 +161,18 @@ public DockerContainer withHealthCheck(Path healthCheckScript)
                 .withCreateContainerCmdModifier(command -> command.withHealthcheck(cmd));
     }
 
+    /**
+     * Marks this container as temporary, which means that it's not expected to be working at the end of environment creation.
+     * Mostly used to execute short configuration scripts shipped as a part of docker images.
+     *
+     * @param temporary
+     */
+    public DockerContainer setTemporary(Boolean temporary)
```

Comment:
just use `boolean` and remove rnn

Reply:
It can throw npe...

Reply:
that is not your problem, it is on callee side. If they use `Boolean` they need to care about case of a null there.

Reply:
Why does this method accept `Boolean` and not `boolean` ?